applications/projects/working-with-projects.adoc

```diff
@@ -8,6 +8,11 @@ toc::[]
 A _project_ allows a community of users to organize and manage their content in
 isolation from other communities.
 
+[NOTE]
+====
+Projects xref:../../authentication/using-rbac.adoc#rbac-default-projects_using-rbac[starting with `openshift-` host clustre components that run as pods and other infrastructure components]. As such, {product-title} does not allow you to create Projects starting with `openshift-` or `kube-` using the `oc new-project` command.
```

Comment:
> clustre

Typo of cluster.

Reply:
@xingxingxia I apologize for this typo. I did make the change. But I didn't save the file before doing the commit. Sorry to waste your time here.